Neos.ContentRepository.Core/Classes/DimensionSpace/InterDimensionalVariationGraph.php

```diff
@@ -127,7 +127,13 @@ public function getRootGeneralizations(): array
                 $rootGeneralizations[$dimensionSpacePointHash] = $weightedDimensionSpacePoint->dimensionSpacePoint;
             }
         }
-
+        if (empty($rootGeneralizations)) {
```

Comment:
nitpick, `empty` is nasty
```suggestion
        if ($rootGeneralizations === []) {
```